docs/content/Configuration/Environment-Variables-Reference.md

```diff
@@ -35,13 +35,13 @@ options][link-config].
 | `CUBEJS_REDIS_POOL_MAX`              | The maximum number of connections to keep active in the Redis connection pool. Must be higher than `CUBEJS_REDIS_POOL_MIN`. Defaults to `1000`                                                   | A valid number of connections.                                                                        |
 | `CUBEJS_REDIS_POOL_MIN`              | The minimum number of connections to keep active in the Redis connection pool. Must be lower than `CUBEJS_REDIS_POOL_MAX`. Defaults to `2`                                                       | A valid number of connections                                                                         |
 | `CUBEJS_REDIS_USE_IOREDIS`           | Use [`ioredis`][gh-ioredis] instead of[ `redis`][gh-node-redis]. Defaults to `false`                                                                                                             | `true`, `false`                                                                                       |
-| `CUBEJS_JWK_KEY`                     | A JSON string that represents a cryptographic key and its' properties                                                                                                                            | A JSON object conforming to [JWK specification RFC 7517 Section 4][link-jwk-ref], encoded as a string |
+| `CUBEJS_JWK_KEYS`                    | A JSON string that contains an array with object where each one represents a cryptographic key and its' properties                                                                               | A JSON array conforming to [JWK specification RFC 7517 Section 4][link-jwk-ref], encoded as a string |
```

Comment:
```suggestion
| `CUBEJS_JWK_KEYS`                    | A JSON string that contains an array of objects where each object represents a cryptographic key and its' properties                                                                               | A JSON array conforming to [JWK specification RFC 7517 Section 4][link-jwk-ref], encoded as a string |
```

Reply:
Not sure if this is correct? Let me know if it isn't 😄

Reply:
Thanks!